spec/acceptance/exec_powershell_spec.rb

```diff
@@ -34,6 +34,31 @@
 
   end
 
+  describe 'should handle a try/catch successfully' do
```

Comment:
The issue I had was that if you had a try catch block ***NOTHING*** was executed at all whatsoever in the script. I think I might agree that without some verification that the actual script ran that this is not a fully baked solution.